samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts

```diff
@@ -1,18 +1,19 @@
 import { Injectable } from '@nestjs/common';
 import {
   BusinessPartner,
-  businessPartnerService,
-} from '@sap/cloud-sdk-vdm-business-partner-service';
+  cloudBusinessPartnerService as businessPartnerService,
+} from '../generated/cloud-business-partner-service';
 const { businessPartnerApi } = businessPartnerService();
 
-const destinationName: string = process.env.CLOUD_DESTINATION;
 @Injectable()
 export class CloudBusinessPartnerService {
   async getFiveBusinessPartners(): Promise<BusinessPartner[]> {
     return businessPartnerApi
       .requestBuilder()
       .getAll()
       .top(5)
-      .execute({ destinationName: destinationName });
```

Comment:
Same here & on the other 2 endpoints

Reply:
yes I have also one more look in the diff for suspicious things.